internal/controller/controllers/agentserviceconfig_controller.go

```diff
@@ -176,15 +181,7 @@ func (r *AgentServiceConfigReconciler) Reconcile(origCtx context.Context, req ct
 		return ctrl.Result{}, err
 	}
 
-	// We only support one AgentServiceConfig per cluster, and it must be called "agent". This prevents installing
-	// AgentService more than once in the cluster.
-	if instance.Name != agentServiceConfigName {
-		reason := fmt.Sprintf("Invalid name (%s)", instance.Name)
-		msg := fmt.Sprintf("Only one AgentServiceConfig supported per cluster and must be named '%s'", agentServiceConfigName)
-		log.Info(fmt.Sprintf("%s: %s", reason, msg), req.NamespacedName)
-		r.Recorder.Event(instance, "Warning", reason, msg)
-		return reconcile.Result{}, nil
-	}
+	log.Info("DEBUG ==> ASC instance is", instance.Name, instance.Namespace)
```

Comment:
I think this log can be removed, but if you do actually want a debug message here, you should use `log.Debug`